packages/ceramic-cli/src/ceramic-cli-utils.ts

```diff
@@ -225,17 +225,7 @@ export class CeramicCliUtils {
      * @param isUnique - Should document be unique?
      */
     static async schemaCreateDoc(content: string, owners: string, onlyGenesis: boolean, isUnique: boolean): Promise<void> {
-        try {
-            const schemaObj = JSON.parse(content)
-            if (!DoctypeUtils.isSchemaValid(schemaObj)) {
-                console.error('Invalid schema')
-                return
-            }
-        } catch (e) {
-            console.error(e.message)
-            return
-        }
-
+        // TODO validate schema on the client side
```

Comment:
without client validation this is the same as `createDoc`. I left it here since I want to add client validation as well :)